.github/workflows/build_packages.yaml

```diff
@@ -16,148 +16,115 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
+              builder:
+                - 5.0-26
               os:
                 - ubuntu20.04
                 - ubuntu18.04
+                - debian11
                 - debian10
-                - debian9
-                - centos8
-                - centos7
+                - el8
+                # Fails with the following message:
+                # The compiler feature "cxx_std_17" is not known to CXX compiler "GNU" version 4.8.5.
+                # - el7
+                - amzn2
               otp:
-                - erl23.2.7.2-emqx-2
-                - erl24.0.1-emqx-1
+                - 24.3.4.2-1
+                - 25.1.2-2
+              elixir:
+                - 1.13.4
 
         steps:
-        - uses: actions/checkout@v1
+        - uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
```

Comment:
why do we want full history?

Reply:
Because the build script [needs][2] some recent git tag, and the checkout action fetches tags only if [`fetch-depth: 0`][1]. Strictly speaking that might be unnecessary because the build script works even without tags, and the checkout action seems smart enough to setup git tag in shallow clones during `release`-triggered workflow runs. I'll remove that line.

[1]: https://github.com/actions/checkout/issues/701#issuecomment-1089864361
[2]: https://github.com/keynslug/emqtt/blob/25905592c6009b31facd0f7a1c34dfd2d09b7535/.github/workflows/script/build.sh#L9-L10

Reply:
Well, that didn't work for a different reason, packaging scripts became confused.
* https://github.com/emqx/emqtt/actions/runs/3836441989/jobs/6530616899#step:3:637
* https://github.com/emqx/emqtt/actions/runs/3836441989/jobs/6530618157#step:3:695